core/src/main/java/org/dozer/BeanFactory.java

```diff
@@ -32,7 +32,8 @@
  *
  * <p>
  * <a
- * href="http://dozer.sourceforge.net/documentation/custombeanfactories.html">http://dozer.sourceforge.net/documentation/custombeanfactories.html</a>
+ * href="https://github.com/DozerMapper/dozer/blob/master/docs/asciidoc/documentation/custombeanfactories.adoc">
```

Comment:
My personal preference is for GitHub pages, because those links will be pretty rendered by browser)